### PR TITLE
balachandra mountain view file addition

### LINKED ISSUE #13: Create a new file for balachandra Mountain View, CA
- [ ] Create a branch
- [ ] Add commits
- [ ] Open a PR
- [ ] Collaborate
- [ ] Seek approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by balachandra**
closes #13

### CODE REVIEW

**Comment by githubteacher** [comment]
👋

File: balachandra-mountainview.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Mountain View, CA
+
```
Comment (githubteacher) [inline balachandra-mountainview.md]: What's the weather like?

Comment (balachandra) [reply balachandra-mountainview.md]: Awesome

Comment (githubteacher) [inline balachandra-mountainview.md]: ⛰ 🌆

**Review by githubteacher — APPROVED** [review]
This is great! Excited to visit mountain view in the future!

**Pull request merged**